apps/docs/content/guides/platform/read-replicas.mdx

```diff
@@ -63,35 +61,61 @@ Projects must meet these requirements to use Read Replicas:
 
 To add a Read Replica, go to the [Infrastructure Settings page](/dashboard/project/_/settings/infrastructure) in your dashboard. Each project can deploy up to two Read Replicas and all Read Replicas inherit the compute size of their Primary database.
 
+### Deploying a Read Replica
+
+A Read Replica is deployed by using a physical backup as a starting point, and a combination of WAL file archives and direct replication from the Primary database to catch up. Both components may take significant time to complete. The duration of restoring from a physical backup is roughly dependent and directly related to the database size of your project. As for catching up with WAL file archives and direct replication, this would be dependent on the amount of WAL files and transactions to be replayed. These are then dependent on the activity of the Primary database. The more active it is, the more WAL files and transactions are generated.
+
+Along with the progress of the deployment, the dashboard displays rough estimates for each component.
+
+### What does it mean when "Init failed" is observed?
+
+The status `Init failed` indicates that the Read Replica has failed to deploy. It is safe to drop this failed Read Replica, and attempt to spin up another one. If however spinning up Read Replicas for your project consistently fails, open a support ticket [here](/dashboard/support/new). To aid the investigation, do not bring down the recently failed Read Replica.
```

Comment:
Would be good to mention some of the transient reasons this could happen (e.g. "this can happen if the RR was initialized while there was a change being made to the primary DB") - right now the wording makes it sound like you're flipping a coin on whether or not things will work